tiledb/sm/array_schema/dimension.h

```diff
@@ -163,9 +163,10 @@ class Dimension {
 
     // Check range bounds
     if (r[0] > r[1]) {
-      *err_msg =
-          "Cannot add range to dimension; Lower range "
-          "bound cannot be larger than the higher bound";
+      std::stringstream ss;
+      ss << "Cannot add range to dimension; Lower range " << r[0]
+         << " bound cannot be larger than the higher bound " << r[1];
```

Comment:
`"... Lower range bound " << r[0]`, i.e., word "bound" before the value for consistency with the "higher bound". Same below :).